packages/root/src/Root.story.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, select } from '@storybook/addon-knobs';
+import Root from '.';
+
+storiesOf('Root', module)
+  .add('Anchor', () => (
```

Comment:
Even though you include Button in the knobs select, I'd add a default state as well. I'd want to make sure all functionality is demonstrated, and then knobs further allows for more introspection.

```suggestion
  .add('Default, () => (
    <Root>
      By default, I am a button.
    </Root>
  ))
  .add('Anchor', () => (
```

Reply:
👍